dashboard/app/views/levels/_form.html.haml

```diff
@@ -66,6 +66,9 @@
 
   var inputs = $('.edit_level *[id]:visible');
   var form_values = {};
+  var TO_SKIP = [
```

Comment:
nit: might be nice to name this a bit more clearly; "FORM_VALUES_TO_SKIP" or something